server/requirements.txt

```diff
@@ -4,10 +4,10 @@ boto3
 click>=8.0
 python-dateutil
 elasticsearch==7.13.1
```

Comment:
I think that this should read
```
elasticsearch>=7.13.1, !=7.14.0
```
Anyone (@dbutenhof?) concur?

Reply:
It's certainly a valid experiment. 7.13.1 was selected simply because it was the version successfully running on my development VM, and the 7.14.0 our jam session install pulled clearly was broken. It's likely but not guaranteed that any 7.14.x or 7.15.x will fix the problem, and at least reasonable to determine whether higher versions of 7.13.x work. I just didn't want to deal with that at the time!

Reply:
Yeah, I tried this and it pulled in 7.15...  I'm hoping that we can determine whether that's viable at tomorrow's Jam session.